remove-git-branch

```diff
@@ -35,13 +35,18 @@ for i in "$@"; do
       FORCE="true"
       shift # past argument=value
       ;;
+    -m|--merged-only)
+      BRANCHES=$(git fetch -p && git branch -v | fgrep '[gone]' | awk '{print $1}' | tr "\n" "," | sed 's/,$/\n/')
+      shift # past argument=value
+      ;;
     -h|--help)
     echo "usage: remove-git-branch  [-b=<pattern> | --branch=<pattern>] [-p=<pattern> | --pattern=<pattern>]
-                [-a | --all] [-h | --help]"
+                [-e | --exclude] [-a | --all] [-m | --merged-only] [-c | --check] [-f | --force] [-h | --help]"
       echo "-b (--branch) - Branches to remove separated by ','. It has priority over pattern."
       echo "-p (--pattern) - Regex pattern to define witch branches should be removed."
-      echo "-a (--all) - Remove all branches, expect master, **main, and current."
       echo "-e (--exclude) - Branches to exclude from deleting."
+      echo "-a (--all) - Remove all branches, expect master, **main, and current."
```

Comment:
Could you also remove `**` from `main`? 🙏 
Not sure why is there, I think it is some leftover.

Reply:
Sure.